### PR TITLE
Add ddc source

### PR DESCRIPTION
**Pull request by Omochice**
This PR provide a source for ddc.vim.

### CODE REVIEW

File: autoload/vsnip_integ/detection.vim [hunk 1]
```diff
@@ -8,6 +8,7 @@ let s:definition = {
 \   'mucomplete': { -> exists('g:loaded_mucomplete') },
 \   'deoplete_lsp': { -> s:runtimepath('lua/deoplete.lua') },
 \   'completion_nvim': { -> exists('g:loaded_completion') },
+\   'ddc': { -> !exists('g:loaded_ddc_vsnip') },
```
Comment (hrsh7th) [inline autoload/vsnip_integ/detection.vim:11]: It seems wrong. It should check the ddc is installed or not.

File: autoload/vsnip_integ/integration/ddc.vim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"
+" vsnip_integ#integration#ddc#attach
+"
+let s:script = expand('<sfile>')
+
+function! vsnip_integ#integration#ddc#attach() abort
+  let g:loaded_ddc_vsnip = 1
```
Comment (hrsh7th) [inline autoload/vsnip_integ/integration/ddc.vim:7]: It isn't needed.

File: denops/ddc/sources/vsnip.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import {
```
Comment (hrsh7th) [inline denops/ddc/sources/vsnip.ts:1]: LGTM!

**Comment by Omochice** [comment]
I fixed some codes pointed.
How are these?

**Comment by hrsh7th** [comment]
LGTM. Thank you~

**Pull request merged**